app/src/test/java/com/simplemobiletools/calculator/MainActivityTest.kt

```diff
@@ -25,146 +23,192 @@ class MainActivityTest {
     @Test
     fun addSimpleDigit() {
         activity.calc.addDigit(2)
-        assertEquals("2", getDisplayedNumber())
+        assertEquals("2", activity.getResult())
```

Comment:
I like this use of getResult. the logic is all in the activity, which we are testing now.